emercoin-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,203 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountNamePrefix": {
```

Comment:
That parameter only for the CI. When a user enters this parameter they are bound to make a mistake if they enter a capital letter or a special char.